comid-triples.cddl

```diff
@@ -29,24 +31,43 @@ endorsed-triple-record = [
   one-or-more<measurement-map> ; endorsed measurements
 ]
 
+; ATTESTATION-VERIFICATION-KEY triple
+; "attesting environment $AE" "signs Evidence that can be verified with key" "$K"
+attest-key-triple-record = [
+  environment-map ; attesting environment
+  one-or-more<verification-key> ; attestation verification key(s)
+]
+
 ; DEVICE-IDENTITY triple
-; (Should it be AE-IDENTITY instead?)
-; "attesting environment $AE" "signs Evidence with key" "$K"
+; "device $D" "is identified by key" "$K"
 identity-triple-record = [
-  environment-map ; attesting-environment
-  one-or-more<COSE_Key> ; identity
+  environment-map ; device identifier (instance or class)
+  one-or-more<verification-key> ; DevID, or semantically equivalent
 ]
 
+verification-key = {
+  ; Verification key (typically a public key)
+  corim.key => COSE_Key
```

Comment:
line 50 usage is it corim.key justified as this variable is also used in Device identity.
Perhaps a generic name would help here like
verif.key ?

Reply:
using `comid.` -- see 43088f8